examples/platform/silabs/SilabsDeviceAttestationCreds.cpp

```diff
@@ -126,6 +120,31 @@ class DeviceAttestationCredsSilabs : public DeviceAttestationCredentialsProvider
 
         return CopySpanToMutableSpan(ByteSpan(signature, signature_size), out_span);
     }
+
+private:
+    CHIP_ERROR GetFileAddress(uint32_t offset_key, uint32_t offset_default, uint8_t *& address)
```

Comment:
since we always deal with a combo of offset + size it might handy to think of an helper that factorize this sanity check to obtain your ByteSpan